### PR TITLE
Introducing Basic HttpMonitoring Widget

### PR DESCRIPTION
**Pull request by amitmawkin**
This replaces the dummy monitoring widget and introduces the basic http monitoring.

The widget currently does not supports proxy and custom self signed certificate needs to be imported manually in the api jvm using the keytool.

Some screen shots below

**Adding a service**

image

**Configured Services**
image

Adding a dependent Service
image

Viewing details of a Service
image

### CODE REVIEW

**Comment by tabladrum** [comment]
@amitmawkin squash?

**Comment by tabladrum** [comment]
@MarkRx please take a look too.

File: pom.xml [hunk 1]
```diff
@@ -220,6 +220,7 @@
         <module>collectors/scm/github</module>
         <module>collectors/scm/subversion</module>
         <module>collectors/scm/gitlab</module>
+          <module>UI</module>
```
Comment (MarkRx) [inline pom.xml]: why is this showing up as an addition?

File: features.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ 
+ `http://samplewebsite.com or https://samplewebsite.com`
+ 
+ Please do remember that if you are using a self signed certificate you would run into following error
+ 
+ `2016-11-14 17:08:25,452 ERROR c.c.d.service.ServiceServiceImpl - https://abc1234.com failed with sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target`
+ 
+ to resolve this please import the certificate into the JVM running your api using java keytool. 
+ 
+ Also at this time, the monitoring widget does not supports any proxy configuration.
```
Comment (MarkRx) [inline features.md]: Why not? Setting up proxy config is pretty straight forward

**Pull request closed without merging**